support-frontend/assets/helpers/forms/paymentIntegrations/payPalRecurringCheckout.ts

```diff
@@ -209,7 +213,7 @@ function setupPayment(
 	billingPeriod: BillingPeriod,
 	setupPayPalPayment: SetupPayPalRequestType,
 ) {
-	return (resolve, reject) => {
+	return (resolve: (arg0: string) => void, reject: (arg0: Error) => void) => {
```

Comment:
It might be worth using some more descriptive names here e.g

`reject: (error: Error) => void`?